test/src/e2e_vm_tests/test_programs/should_pass/test_abis/contract_abi_auto_impl/Forc.toml

```
@@ -0,0 +1,8 @@
+[project]
+name = "contract_abi_auto_impl"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "main.sw"
+license = "Apache-2.0"
+
+[dependencies]
+std = { path = "../../../../reduced_std_libs/sway-lib-std-assert" }
```

You're not using std in the test, just don't include it.
```suggestion
implicit-std = false
```